addons/google_calendar/models/calendar_attendee.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+
+class CalendarAttendee(models.Model):
+    _inherit = 'calendar.attendee'
+
+    google_internal_event_id = fields.Char('Google Calendar Event Id')
+    date_synchro = fields.Datetime('Odoo Synchro Date')
+
+    _sql_constraints = [('google_id_uniq', 'unique(google_internal_event_id,partner_id,event_id)', 'Google ID should be unique!')]
+
+    @api.multi
+    def write(self, vals):
+        for record in self:
+            ref = vals.get('event_id', record.event_id.id)
```

Comment:
use event_id variable name instead of ref. so we can easily identify it's contain event id ..